src/reconcile.py

```diff
@@ -61,8 +56,36 @@ def _get_server_content(table_row: types_.TableRow, discourse: Discourse) -> str
         ) from exc
 
 
+def _local_and_server_validation(
+    path_info: types_.PathInfo,
+    table_row: types_.TableRow,
+) -> None:
+    """Input checks before execution.
+
+    Args:
+        path_info: Information about the local documentation file.
+        table_row: A row from the navigation table.
+
+    Raises:
+        ReconcilliationError:
```

Comment:
Also raises RepositoryClientError

Reply:
Where do you see that?

Reply:
I don't see it anymore with the new changes. Now that exception is always catch and re-raised